lib/netsparker/vulnerability.rb

```diff
@@ -147,6 +147,9 @@ def cleanup_html(source)
 
       result.gsub!(/<a .*?href=(?:\"|\')(.*?)(?:\"|\').*?>(?:<i.*?<\/i>)?(.*?)<\/a>/i) { "\"#{$2.strip}\":#{$1.strip}" }
 
+      result.gsub!(/<code>(.*?)<\/code>/) { "@#{$1}@" }
+      result.gsub!(/<pre.*?>(.*?)<\/pre>\n/) { "\n\nbc. #{$1}\n\n" }
```

Comment:
Line 154 already catches the `<pre>` tag. Instead of adding this one I'm thinking Line 154 should be updated instead to accommodate this change

Reply:
Good catch, thanks @aapomm ! I tested and confirmed that we can remove that line entirely and pushed the updated version.